_posts/0000-01-02-rahmansk.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This one looks good !!!

![download](https://user-images.githubusercontent.com/62998470/163708471-9dc2916f-b6b8-465a-a658-aa08be54fc37.gif)